### PR TITLE
14.0 sale partner version imp

### PR DESCRIPTION
**Pull request by kevinkhao**

### CODE REVIEW

**Comment by kevinkhao** [comment]
Depends on https://github.com/OCA/partner-contact/pull/1085

**Comment by kevinkhao** [comment]
@sebastienbeau

**Comment by kevinkhao** [comment]
this branch needs https://github.com/akretion/sale-workflow/pull/23 to go green

File: sale_partner_version/models/res_partner.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+from odoo import models
+
+
+class ResPartner(models.Model):
+    _inherit = "res.partner"
+
+    def _version_impacted_columns(self):
+        result = super()._version_impacted_columns()
+        result += [
+            ("sale_order", "partner_id"),
```
Comment (sebastienbeau) [inline sale_partner_version/models/res_partner.py:13]: I do not think that we should archive the partner_id, I prefers to keep it on the active contact

**Pull request closed without merging**